packages/atomic/src/components/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -29,6 +29,12 @@ export class AtomicSearchBoxRecentQueries {
 
   @State() public error!: Error;
 
+  /**
+   * The icon that will be passed down to `atomic-icon` to be shown next to each suggestion on the list.
+   * Defaults to atomic icon. Setting it to an empty string hides it.
+   */
+  @Prop() public icon?: string;
```

Comment:
It shows/hides itself when there other types of suggestions along with it. It's what UX deemed best, there should always be at least a default icon

Reply:
alright